From 8efda1fd89ac85741f1d9196ca66444010ffc740 Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Sat, 4 Oct 2025 23:43:01 +0200
Subject: [PATCH 76/98] Settings: Switch back to
 udfps_enroll_enrolling_non_scroll layout

New one seems halfbaked? The UDFPS icon is misaligned when using it and
progress circle is missing.

Change-Id: I19d0a1ff973d79381f0f315469c823afedde9c02
---
 .../FingerprintEnrollEnrolling.java           | 33 +------------------
 1 file changed, 1 insertion(+), 32 deletions(-)

diff --git a/src/com/android/settings/biometrics/fingerprint/FingerprintEnrollEnrolling.java b/src/com/android/settings/biometrics/fingerprint/FingerprintEnrollEnrolling.java
index 80e285ef112..f68b836c72b 100644
--- a/src/com/android/settings/biometrics/fingerprint/FingerprintEnrollEnrolling.java
+++ b/src/com/android/settings/biometrics/fingerprint/FingerprintEnrollEnrolling.java
@@ -249,30 +249,12 @@ public class FingerprintEnrollEnrolling extends BiometricsEnrollEnrolling {
 
             final UdfpsEnrollEnrollingView layout =
                     (UdfpsEnrollEnrollingView) getLayoutInflater().inflate(
-                            Flags.enrollLayoutTruncateImprovement()
-                                    ? R.layout.udfps_enroll_enrolling :
-                                    R.layout.udfps_enroll_enrolling_non_scroll, null, false);
+                            R.layout.udfps_enroll_enrolling_non_scroll, null, false);
             setUdfpsEnrollHelper();
             layout.initView(props.get(0), mUdfpsEnrollHelper, mAccessibilityManager);
             setContentView(layout);
             setDescriptionText(R.string.security_settings_udfps_enroll_start_message);
 
-            if (Flags.enrollLayoutTruncateImprovement() && isPortrait) {
-                final UdfpsEnrollEnrollingView layoutView = (UdfpsEnrollEnrollingView) getLayout();
-                if (layoutView != null) {
-                    final ScrollView headerScrollView = layout.findViewById(
-                            R.id.sud_header_scroll_view);
-                    if (headerScrollView != null) {
-                        final long headerScrollDuration = getResources().getInteger(
-                                R.integer.config_biometrics_header_scroll_duration);
-                        layoutView.adjustScrollableHeaderHeight(
-                                headerScrollView, mShouldShowLottie);
-                        layoutView.headerVerticalScrolling(headerScrollView, headerScrollDuration,
-                                mIsAccessibilityEnabled);
-                    }
-                }
-            }
-
         } else if (mCanAssumeSfps) {
             mSfpsEnrollmentFeature = FeatureFactory.getFeatureFactory()
                     .getFingerprintFeatureProvider().getSfpsEnrollmentFeature();
@@ -1236,19 +1218,6 @@ public class FingerprintEnrollEnrolling extends BiometricsEnrollEnrolling {
     @Override
     public void onWindowFocusChanged(boolean hasWindowFocus) {
         super.onWindowFocusChanged(hasWindowFocus);
-        if (Flags.enrollLayoutTruncateImprovement()) {
-            adjustEnrollViewIfOverlappedWithFooterBar();
-        }
-    }
-
-    private void adjustEnrollViewIfOverlappedWithFooterBar() {
-        if (mCanAssumeUdfps) {
-            final UdfpsEnrollEnrollingView layoutView = (UdfpsEnrollEnrollingView) getLayout();
-            if (layoutView != null) {
-                layoutView.adjustUdfpsVieWithFooterBar();
-                layoutView.onUdfpsSensorRectUpdated();
-            }
-        }
     }
 
     public static class IconTouchDialog extends InstrumentedDialogFragment {
